### PR TITLE
"fix"/hack

### PR DESCRIPTION
**Pull request by trzysiek**
Ideally I wouldn't like to merge any of this, maybe besides size being string, but I'm afraid most or all of this was needed for all dashboards to work. I'll need to investigate.

### CODE REVIEW

File: quesma/queryparser/query_parser.go [hunk 1]
```diff
@@ -1262,6 +1262,13 @@ func (cw *ClickhouseQueryTranslator) parseSize(queryMap QueryMap, defaultSize in
 		return defaultSize
 	} else if sizeAsFloat, ok := sizeRaw.(float64); ok {
 		return int(sizeAsFloat)
+	} else if sizeAsString, ok := sizeRaw.(string); ok {
```
Comment (mieciu) [inline quesma/queryparser/query_parser.go:1265]: I'd suggest moving this to separate PR and merging to `main`

Comment (mieciu) [reply quesma/queryparser/query_parser.go:1265]: By "this" I mean the whole `quesma/queryparser/query_parser.go` change of course 😉

**Comment by trzysiek** [comment]
No need for it, it even stopped passing tests.
